packages/react/src/MinimalFooter/MinimalFooter.test.tsx

```diff
@@ -0,0 +1,478 @@
+import React from 'react'
+import {render} from '@testing-library/react'
+import '@testing-library/jest-dom'
+import {axe, toHaveNoViolations} from 'jest-axe'
+import {MinimalFooter} from './MinimalFooter'
+import {Text} from '../'
+
+expect.extend(toHaveNoViolations)
+
+// Helper function to filter social links
+const getSocialLinks = (links: HTMLElement[]) => {
+  return links.filter(
+    link =>
+      link.getAttribute('href')?.includes('x.com') ||
+      link.getAttribute('href')?.includes('github.com/github') ||
+      link.getAttribute('href')?.includes('linkedin.com') ||
```

Comment:
Can we do an equality check instead of a partial match? Also wondering if we can just duplicate the same `socialLinkData` here instead, as that would give better confidence.